samples/dotnet/KernelHttpServer/Plugins/GitHubPlugin.cs

```diff
@@ -196,10 +197,10 @@ private async Task SummarizeCodeFileAsync(string filePath, string repositoryUri,
             else
             {
                 await this._kernel.Memory.SaveInformationAsync(
-                    $"{repositoryUri}-{repositoryBranch}",
-                    text: $"{code} File:{repositoryUri}/blob/{repositoryBranch}/{fileUri}",
-                    id: fileUri,
-                    cancellationToken: cancellationToken);
+                     $"{repositoryUri}-{repositoryBranch}",
+                     text: $"{code} File:{repositoryUri}/blob/{repositoryBranch}/{fileUri}",
+                     id: fileUri,
+                     cancellationToken: cancellationToken);
```

Comment:
nit: get rid of the extra space at the beginning of each line? (unless that's an intentional spacing fix)

Reply:
I thought visual studio was fix bading spacing...either way, I didn't mean to do this.  I've repeatedly auto-formatted...manually fixing.